tasks/test.py

```diff
@@ -33,26 +34,55 @@ def ensure_bytes(s):
     return s
 
 
+@contextmanager
+def environ(env):
+    original_environ = os.environ.copy()
+    os.environ.update(env)
+    yield
```

Comment:
I am a bit puzzled by this `yield` here - what is it supposed to do?

Reply:
`environ` is to be used as a context manager; the part above the yield is the setup, the part below it os the cleanup.

Reply:
I would probably guess that we want a try/except/finally pattern around the yield then to ensure we clean up things but it's not a blocker.